EventsByTag.md

```diff
@@ -85,6 +85,14 @@ In this case the eventsByTag query looks for the new-persistence-id-scan-timeout
 is found by an offset query. If this is an issue it can be set to 0s. If events are found out of order due to this
 the stage will fail.  
 
+### Delayed events and out of sync clocks
+
+Events for a tag are globally ordered using a TimeUUID.
+However events for the same tag but for a difference persistenceIds can come from different nodes in a cluster meaning that
+events enter Cassandra out of TimeUUID. If a live eventsByTag query is currently querying it will see these events out of order. For
+that reason an `eventual-consistency-delay` is used to keep eventsByTag a configurable amount of time in the past to give a time
+for events from all nodes to settle to their final order.
```

Comment:
Can we make it more clear what the consequences of reducing or disabling the ec delay are? I think most users are fine with that the order is only guaranteed per persistenceId, but are there other consequences than not receiving all events in "global" time order? Relation to missing events section?